string personality
---
float64[] params
